apps/dashboard/src/app/drops/[slug]/page.tsx

```diff
@@ -0,0 +1,134 @@
+import { getThirdwebClient } from "@/constants/thirdweb.server";
+import { defineDashboardChain } from "lib/defineDashboardChain";
+import { notFound } from "next/navigation";
+import { getContract, toTokens } from "thirdweb";
+import { getContractMetadata } from "thirdweb/extensions/common";
+import { getCurrencyMetadata } from "thirdweb/extensions/erc20";
+import {
+  getActiveClaimCondition as getActiveClaimCondition721,
+  getNFT as getNFT721,
+} from "thirdweb/extensions/erc721";
+import {
+  getActiveClaimCondition as getActiveClaimCondition1155,
+  getNFT as getNFT1155,
+} from "thirdweb/extensions/erc1155";
+import { NftMint } from "./mint-ui";
+
+type DropPageData = {
+  slug: string;
+  contractAddress: string;
+  chainId: number;
+  hideQuantitySelector?: boolean;
+  hideMintToCustomAddress?: boolean;
+  // If not defined, we will use the image of the NFT or contract's image
+  thumbnail?: string;
+} & ({ type: "erc1155"; tokenId: bigint } | { type: "erc721" });
+
+export const DROP_PAGES: DropPageData[] = [
```

Comment:
Remove export here - exporting non-next things from page.tsx is not valid

Reply:
no way I didn't know this!